src/main/java/ua/edu/internship/interview/service/business/SkillService.java

```diff
@@ -0,0 +1,30 @@
+package ua.edu.internship.interview.service.business;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+import ua.edu.internship.interview.service.dto.SkillTreeDto;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChA2NTUwNWJhNWFlMmZhMDNkEgpjaGVja3N0eWxl -->